src/rust/engine/src/externs/testutil.rs

```diff
@@ -41,7 +41,7 @@ impl PyStubCASBuilder {
     Ok(PyStubCASBuilder(self.0.clone()))
   }
 
-  fn build(&mut self, py_executor: PyExecutor) -> PyResult<PyStubCAS> {
+  fn build(&mut self, py_executor: &PyExecutor) -> PyResult<PyStubCAS> {
```

Comment:
pyo3 will use `Clone` to pass objects by value if requested, but we removed `Clone` from `PyExecutor` here to make `Drop` meaningful.